test/soca/create_obsdb.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+import argparse
+import os
+from r2d2 import store
+from solo.configuration import Configuration
+from solo.date import date_sequence
+import yaml
+
+
+def store_obs(yaml_file):
+    config = Configuration(yaml_file)
+    dates = date_sequence(config.start, config.end, config.step)
+    obs_types = config.obs_types
+    provider = config.provider
+    experiment = config.experiment
+    database = config.database
+    type = config.type
+    source_dir = config.source_dir
+    step = config.step
+
+    for date in dates:
+        day = str(date).split('T')[0]
+        year = day[0:4]
+        month = day[4:6]
+        day = day[6:8]
+        for obs_type in obs_types:
+            obs_prefix = obs_type.split('_')[0]
+            store(
+                provider=provider,
+                type=type,
+                experiment=experiment,
+                database=database,
+                date=date,
+                obs_type=obs_type,
+                time_window=step,
+                source_file=f'{source_dir}/{obs_type}_{year}{month}{day}.nc4',
+                ignore_missing=True,
+            )
+
+
+if __name__ == "__main__":
+
+    # Set the R2D2_CONFIG environement variable
+    r2d2_config = {'databases': {'archive': {'bucket': 'archive.jcsda',
+                                             'cache_fetch': True,
+                                             'class': 'S3DB'},
+                                 'local': {'cache_fetch': False,
+                                           'class': 'LocalDB',
+                                           'root': './r2d2-local/'},
+                                 'shared': {'cache_fetch': False,
+                                            'class': 'LocalDB',
+                                            'root': './r2d2-shared/'}},
+                   'fetch_order': ['local'],
+                   'store_order': ['shared', 'local', 'archive']}
+    f = open('r2d2_config_test.yaml', 'w')
+    yaml.dump(r2d2_config, f, sort_keys=False, default_flow_style=False)
+    os.environ['R2D2_CONFIG'] = 'r2d2_config_test.yaml'
+
+    # Change the obs file format
+    os.system('cp -L adt.nc adt_j3_20180415.nc4')
```

Comment:
can these be instead `shutil.copyfile(src, dest, follow_symlinks=True)`?

Reply:
Weird, it works fine on my desktop.

Reply:
It must be a `pwd` issue or something. I wonder if adding `./` will fix it?

Reply:
This is fascinating... Part of me says 'just go back to what was working', but now I'm intrigued as to what the error is here.

Reply:
We have this option ``-DBUILD_GDASBUNDLE=OFF`` for the github ci testing, so whatever I did can't work! My test is copying links with none existent targets. Oops! Somebody with a "slight" french accent might have been in the dumb zone again. Not sure why `os` didn't complain.
@CoryMartin-NOAA I'm going to exclude that test from the github ci testing.

Reply:
Ha, I hadn't noticed that either :-)
I would like R2D2 unit tests to run in GitHub. We can add these files to my .tar.gz that I download for the increment test if you'd like? Otherwise I will just wait until I have one for atm.

Reply:
@CoryMartin-NOAA sounds good.